### PR TITLE
atx.intro1.menchaca.diego

### PR DESCRIPTION
**Pull request by dmencha**

### CODE REVIEW

File: 02LessonTwo/in-class-project/index.html [hunk 1]
```diff
@@ -1,10 +1,52 @@
...
+    <li>Eraserhead</li>
+    <li>Gorilla Mocha</li>
+    <li>Hideout Hazelnut</li>
+    <li>Raspberry Riot</li>
+    <li>Ying-Yang</li>
+</ul>
+<h2>Specials</h2>
+<p>Join us for Happy Hour every day from 5 – 7pm and all night Wednesday! $3 beer and $5 for all of our wine and specialty craft cocktails.</p>
 
+<table>
```
Comment (e-zambrano) [inline 02LessonTwo/in-class-project/index.html:25]: great use of the table element

**Comment by e-zambrano** [comment]
@dmencha - nicely done

File: 02LessonTwo/homework/index.html [hunk 2]
```diff
@@ -1,10 +1,35 @@
...
+<section>
+  <h3>Favorite Websites</h3>
+    <a href="http://www.theverge.com/"target=_blank><img src="http://cdn3.vox-cdn.com/uploads/chorus_asset/file/2404530/logo-theverge.0.svg" alt="The Verge" width="200"></a>
+    <br>
+    <a href="http://jalopnik.com/"target=_blank><img src="https://i.kinja-img.com/gawker-media/image/upload/s--_YRFHT3p--/c_fit,fl_progressive,h_64,q_80/epakhukodybwslfhgrm9.png" alt="Jalopnik" width="200"></a>
+    <br>
+    <a href="https://theringer.com"target=_blank><img src="https://pbs.twimg.com/media/CbbIY05UMAAldK0.jpg:large" alt="The Ringer" width=200></a>
+</section>
+<footer>
+  You can contact me <a href="mailto:dmencha@gmail.com">here</a>.
```
Comment (e-zambrano) [inline 02LessonTwo/homework/index.html]: you are missing your closing element for **footer**

File: 03LessonThree/in-class-project/daily-planet.html [hunk 3]
```diff
@@ -1,10 +1,37 @@
...
+        </nav>
+    </header>
+    <main>
+        <section>
+            <article>
+              <h3>Pilot was able to keep flying despite drunken driving record</h3>
+                LOCKHART, Texas — The pilot of a hot air balloon that crashed in Texas, killing 16 people, was able to keep flying despite having at least four convictions for drunken driving in Missouri and twice spending time in prison. Whether the pilot's drinking
+                habits had anything to do with the crash was unclear. A former girlfriend described Alfred "Skip" Nichols as a recovering alcoholic. She said he had been sober for at least four years and never piloted a balloon after drinking.
+            </article>
+            <img src=http://media.cmgdigital.com/shared/lt/lt_cache/thumbnail/600/img/photos/2016/08/01/c3/f3/d5d935c41e7047abae8877957153daa1-ac8f04d32791412cb88695503978fb02-4.jpg />
```
Comment (e-zambrano) [inline 03LessonThree/in-class-project/daily-planet.html]: remember that your values for any attribute need to be within quotes src="..."

**Comment by e-zambrano** [comment]
@dmencha - I like your progress, just pay attention to the small details before submitting

File: 04LessonFour/in-class-project/css/style.css [hunk 4]
```diff
@@ -1,84 +1,34 @@
 body {
-    background-color: skyblue;
+    background-color: white;
+    width:80%;
+    margin:auto;
+    font-family: Sans-serif;
```
Comment (e-zambrano) [inline 04LessonFour/in-class-project/css/style.css:5]: i think it's `sans-serif`

File: 05LessonFive/Checkpoint1/css/style.css [hunk 5]
```diff
@@ -1,43 +1,31 @@
...
+section {
+  padding-left: 5px;
+  padding-top: 5px;
 }
-
-.left h1 {
+td {
     text-align: center;
+    background-color: #F0F0F0;
+    float: auto;
```
Comment (e-zambrano) [inline 05LessonFive/Checkpoint1/css/style.css:15]: the `float` property does not accept `auto` as a value. remember that we review only 3 possible values

File: 07LessonSeven/homework/index.html [hunk 6]
```diff
@@ -1,10 +1,36 @@
...
+                Lucas ipsum dolor sit amet organa dooku hutt chewbacca han organa watto darth darth. Kamino solo amidala windu. Jawa solo bespin moff
+            </p>
+            <button type="company">Sign up as company</button>
+        </div>
+        <div class="black">
+            <h3>Looking for a job</h3>
+            <p>
+                Lucas ipsum dolor sit amet organa dooku hutt chewbacca han organa watto darth darth. Kamino solo amidala windu. Jawa solo bespin moff
+            </p>
+            <button type="employee">Sign up as employee</button>
```
Comment (e-zambrano) [inline 07LessonSeven/homework/index.html:31]: i am not sure that `type=employee"` it's correct. 
Look [here](http://www.w3schools.com/tags/att_button_type.asp) to see what `type`s are available

**Comment by e-zambrano** [comment]
@dmencha - You are doing an excellent work. 
I noticed a few things that can be improved in some of your CSS rules, but getting better at it will come with practice

**Comment by e-zambrano** [comment]
@dmencha - I am having some trouble getting to exercise and homework from day 12, also i couldn't see your exercise for day 9.
For the rest, you are doing great.

**Comment by dmencha** [comment]
Hey Erik,

I synced these again in Github, I don't think I had properly loaded them.

Diego Menchaca

**Comment by e-zambrano** [comment]
@dmencha - take a close look to the HTML template I sent over slack, I noticed that in some of your html files you had the CSS in the wrong order

File: 07LessonSeven/in-class-project/index.html [hunk 7]
```diff
@@ -2,9 +2,61 @@
...
+                <br> Facebook - fb.com/nasa
+                <br> Instagram - @nasa
+                <br> Youtube - youtube.com/NASA
+                <br>
+            </p>
+        </div>
+        <div class="office">
+            <h3>
+      OFFICE INFO
+    </h3>
```
Comment (CLofton) [inline 07LessonSeven/in-class-project/index.html:31]: nice indentation! the h3 should be moved over a bit.

**Pull request closed without merging**